RELEASING.md

```diff
@@ -331,6 +332,22 @@ This SDK is compatible with 🚙 Engine + 🚗 CLI version `$ENGINE_VERSION`
 - [ ] Click on **Publish release**
 
 
+## 🛝 Playground
+
+The [Dagger Playground](https://play.dagger.cloud) is set to automatically update once there's a new release of the Dagger Engine. 
+In order to verify which version the playground is using, an `x-dagger-engine` HTTP header with the deployed Dagger Engine version is
+returned for each playground query. 
+
+Follow these steps to retrieve and verify the playground version:
+
+1. Browse https://play.dagger.cloud and log-in using GitHub credentials.
+2. Open your developer tools and go to the "Network" tab.
+3. Press the "Execute" button in the UI to send a query.
+4. Click in the `/playgrounds` POST request row in the network tab and take onte of the engine commit version.
+5. Verify that the returned commit version matches with the Engine tag commit generated at the beginning of this guide.
```

Comment:
Tested on my own and get the same output as you got :rocket:

I'm just wondering, why it is a `devel` tag and not a stable one like `v4.0.1`?

Reply:
> I'm just wondering, why it is a `devel` tag and not a stable one like `v4.0.1`?

because we're actually building the CLI in the playground servers since sometimes we need to apply a hotfix so the process basically checks out the code a the required commit and then performs a `go build` on it. That's mostly the reason you're seeing it's `devel` mode.